hathor/feature_activation/bit_signaling_service.py

```diff
@@ -53,7 +53,11 @@ def __init__(
 
         self._validate_support_intersection()
 
-    def start(self) -> None:
+    def generate_initial_logs(self) -> None:
```

Comment:
Even though I understand the expected pre-conditions to run this method, I think that it should be named `start()`.

Reply:
Ok, changed it back in https://github.com/HathorNetwork/hathor-core/pull/828/commits/366e6ae39b96da97bd98bc11fc20f47eef39de4b

Reply:
Thanks!